src/python_testing/TC_IDM_4_2.py

```diff
@@ -385,41 +427,67 @@ async def test_TC_IDM_4_2(self):
         )
 
         # Verify that the subscription is activated between CR1 and DUT
-        asserts.assert_true(sub_cr1_provided_dvf.subscriptionId, "Subscription not activated")
+        asserts.assert_true(sub_cr1_step8.subscriptionId, "Subscription not activated")
 
-        sub_cr1_provided_dvf.Shutdown()
+        sub_cr1_step8.Shutdown()
 
         # *** Step 8 ***
-        self.print_step(8, "CR1 sends a subscription request action for an attribute and sets the MinIntervalFloor value to be same as MaxIntervalCeiling. Activate the Subscription between CR1 and DUT. Modify the attribute which has been subscribed to on the DUT.")
+        self.print_step(8, "CR1 sends a subscription request action for an attribute and sets the MinIntervalFloor to min_interval_floor_sec and MaxIntervalCeiling to 10. Activate the Subscription between CR1 and DUT and record the time when the priming ReportDataMessage is received as t_report_sec. Save the returned MaxInterval from the SubscribeResponseMessage as max_interval_sec.")
 
         # Subscribe to attribute
-        same_min_max_interval_sec = 3
+        min_interval_floor_sec = 3
```

Comment:
```suggestion
        min_interval_floor_sec = 10
```

Reply:
can't do this and it doens't match the test plan. If you do floor == ceiling the device is being asked to send a message before 10 seconds, but also not before 10 seconds.

Reply:
My mistake - misread the test steps